app/models/data_resources/news_item.rb

```diff
@@ -85,7 +85,10 @@ def find_or_create_category
     end
 
     def send_push_notification
+      # do not send push notifications, if not explicitly requested
       return unless push_notification.to_s == "true"
+      # do not send push notification, if already sent
+      return if push_notifications_sent_at.present?
```

Comment:
> Push Notifications bei NewsItem werden bisher :after_create versendet. wenn es aber keine :create mehr gibt sondern nur noch ein Update muss hier der Callback noch angepasst werden

habe es angepasst auf `:after_save`, damit push notifications sowohl bei anlegen als auch aktualisieren erfolgen können. hier habe ich zusätzlich berücksichtigt, dass keine push notifications rausgehen, wenn bereits für eine news erfolgt, was wir anhand des zeitstempels erkennen können.

> hier sind glaube noch nicht alle Folgen abgeklärt, wenn es dann kein create sondern nur noch updates bei bestehenden Daten angeht. Vor allem im Hinblick auf Push Notifications muss hier der Ablauf noch einmal geprüft werden und das Zusammenspiel mit dem CMS

werde als nächstes lokal zusammen mit cms tests anstellen.

Reply:
PR CMS für News Items: https://github.com/ikuseiGmbH/smart-village-app-cms/pull/128
Wenn damit alles in Ordnung ist, gehe ich die übrigen Ressourcen im CMS an.

Wie sieht es aus mit den Importern? Verhalten die sich ohne Änderungen richtig aufgrund der `unique_id`s? Welche Importer "updaten" überhaupt statt kompletten neuen Datensätzen? Wie gehen wir hier sicher?

Gibt es neben CMS und Importer weitere zu berücksichtigende Anwendungen?

Reply:
Wir haben alles nochmal durchgesprochen und haben keine besonderen Fälle ausmachen können.